scripts/package-cli

```diff
@@ -8,13 +8,32 @@ cd $(dirname $0)/..
 GO=${GO-go}
 
 for i in containerd crictl kubectl k3s-agent k3s-server k3s-token k3s-etcd-snapshot k3s-secrets-encrypt k3s-certificate k3s-completion; do
-    rm -f bin/$i
-    ln -s k3s bin/$i
+    rm -f bin/$i${BINARY_POSTFIX}
+    ln -s k3s${BINARY_POSTFIX} bin/$i${BINARY_POSTFIX}
 done
 
-for i in bandwidth bridge firewall flannel host-local loopback portmap; do
-    rm -f bin/$i
-    ln -s cni bin/$i
+cni_binaries=(
+    "bandwidth"
+    "bridge"
+    "firewall"
+    "flannel"
+    "host-local"
+    "loopback"
+    "portmap"
+)
+
+if [ ${OS} = windows ]; then
+    cni_binaries=(
+        "win-bridge"
```

Comment:
Do we need the win-bridge cni plugin? This PR delegates to win-overlay

Reply:
no, I can remove it.